libs/testing/fixtures/src/lib/chart/generate-area-range-data.ts

```diff
@@ -14,24 +14,28 @@
  * limitations under the License.
  */
 
-import { DataPoint } from 'highcharts';
+import { PointOptionsObject, SeriesArearangeDataOptions } from 'highcharts';
 
 import { randomize } from './randomize';
 
+// TODO: find a way without lint exceptions
+// tslint:disable-next-line: nx-enforce-module-boundaries
```

Comment:
Please don't use the `isDefined` function here!
**This rule is very important to keep. You introduce a circular with this import.**


Please duplicate the code for `isDefined` and create an issue that should solve the code duplication without introducing a circular import structur.

Reply:
Thank you, issue created.